app/src/main/java/com/epfl/drawyourpath/database/Database.kt

```diff
@@ -14,6 +21,14 @@ abstract class Database {
      */
     abstract fun isUserNameAvailable(userName: String): CompletableFuture<Boolean>
```

Comment:
Inconsistency in casing. Should be `isUsernameAvailable()` to match other function names.